components/post-text/PostText.tsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import Link from 'next/link';
+import { uniq as _uniq } from 'lodash';
+
+interface IProps {
+  text: string;
+}
+
+const PostText = ({ text }: IProps) => {
+  const regexExp = /#[\p{L}\p{M}0-9_]+/gu;
+
+  const getAndRenderHashTags = (text: string) => {
```

Comment:
You might want to split this function to getHashTags and renderHashTags to meet the Single responsibility principle of SOLID.